README.md

```diff
@@ -24,7 +24,7 @@ sourceControl {
 
 // build.gradle:
 // use the name of a specific tag from https://github.com/dynatrace-oss/opentelemetry-metric-java/tags
-def dynatraceMetricsExporterVersion = "v1.0.0"
+def dynatraceMetricsExporterVersion = "v1.1.0"
```

Comment:
Oops - good catch! 🙂

Reply:
😊

Reply:
Do we need to bump the version here? We're just updating the tests, right? :thinking: